app/assets/stylesheets/layouts/_devise.pcss

```diff
@@ -0,0 +1,42 @@
+.layout-devise {
+  body {
+    @apply h-screen bg-gray-900;
+
+    background: {
+      image: url('../images/background.png');
+      repeat: no-repeat;
+      position: center;
+      size: cover;
+    }
+  }
+
+  .app-main {
+    @apply h-full w-full grid;
+
+    place-items: center;
+    background: linear-gradient(180deg, var(--color-white-200) -50%, var(--color-original-black) 100%);
+    mix-blend-mode: normal;
+    backdrop-filter: blur(40px);
+
+    &__container {
```

Comment:
Why not using `container` ([docs](https://tailwindcss.com/docs/container))?

Reply:
In the design, the width is not too wide, so I create a new element to limit the width